deploy.sh

```diff
@@ -1,7 +1,7 @@
 #!/bin/sh
 
 openstack overcloud deploy --stack overcloud  --working-dir ~/overcloud-deploy/overcloud \
-	  --templates ~/tripleo-heat-templates \
+	  --templates \
 	  -e ~/ironic-overcloud.yaml \
```

Comment:
I just realized these will always source the values in ~, not in the relative directory; we probably want the relative directory